tests/sftp/triggers/test_sftp.py

```diff
@@ -128,12 +137,12 @@ async def test_sftp_with_newer_then_date_greater(self, mock_mod_time):
         asyncio.get_event_loop().stop()
 
     @pytest.mark.asyncio
-    @mock.patch("astronomer.providers.sftp.hooks.sftp.SFTPHookAsync.get_file_by_pattern")
-    async def test_sftp_trigger_run_trigger_failure_state(self, mock_get_file_by_pattern):
+    @mock.patch("astronomer.providers.sftp.hooks.sftp.SFTPHookAsync.get_files_by_pattern")
+    async def test_sftp_trigger_run_trigger_failure_state(self, get_files_by_pattern):
```

Comment:
May I know why do we change it to `get_files_by_pattern` here. According to other test cases in this test file, it seems adding a `mock_` before the mocked function is the convention.

Reply:
added `mock_` .